tests/suites/0_stateless/02_function/02_0000_function_aggregate_argmin_argmax.sql

```diff
@@ -1,10 +1,10 @@
-SELECT sum(number) AS salary, number%3 AS user FROM numbers_mt(10000) GROUP BY user ORDER BY salary ASC;
+SELECT sum(number) AS salary, number%3 AS user_name FROM numbers_mt(10000) GROUP BY user_name ORDER BY salary ASC;
```

Comment:
`USER` is a reserved ident in postgresql. I adopted it here.

Reply:
@BohuTANG should we keep `user` reserved?

Reply:
Ok